relay-server/src/actors/store.rs

```diff
@@ -374,8 +385,24 @@ impl StoreForwarder {
     }
 
     fn send_metric_message(&self, message: MetricKafkaMessage) -> Result<(), StoreError> {
+        let topic = match message.name.parse() {
+            Ok(MetricMri { namespace, .. }) if namespace == "transactions" => {
+                KafkaTopic::MetricsTransactions
+            }
+            Ok(MetricMri { namespace, .. }) if namespace == "sessions" => {
+                KafkaTopic::MetricsSessions
+            }
+            _ => {
```

Comment:
After introducing the explicit "custom" namespace, produce that to the default `Metrics` topic.

Reply:
That's not what we agreed on when we wrote the story fwiw